hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/TimelineServer.md

```diff
@@ -833,13 +833,13 @@ Use the following URI to obtain the event objects of the given `entityType`.
 
 1. `entityId` - The entity IDs to retrieve events for. If null, no events will be returned.
   Multiple entityIds can be given as comma separated values.
-1. `limit` - A limit on the number of events to return for each entity. If null,
+2. `limit` - A limit on the number of events to return for each entity. If null,
   defaults to 100 events per entity.
-1. `windowStart` - If not null, retrieves only events later than the given time
+3. `windowStart` - If not null, retrieves only events later than the given time
   (exclusive)
-1. `windowEnd` - If not null, retrieves only events earlier than the given time
+4. `windowEnd` - If not null, retrieves only events earlier than the given time
   (inclusive)
-1. `eventType` - Restricts the events returned to the given types. If null,
+5. `eventType` - Restricts the events returned to the given types. If null,
```

Comment:
I doubt if this is required. Pass me the link from here where it shows messed up
http://apache.github.io/hadoop